docs/core/native-interop/expose-components-to-com.md

```diff
@@ -80,14 +80,24 @@ Open an elevated command prompt and run `regsvr32 ProjectName.comhost.dll`. That
 
 The resulting output will now also have a `ProjectName.X.manifest` file. This file is the side-by-side manifest for use with Registry-Free COM.
 
+## Embedding Type Libraries in the COM host
+
+Unlike in .NET Framework, there is no support in .NET Core or .NET 5 or newer for generating a COM Type Library (TLB) from a .NET Core assembly. The guidance is to either manually write an IDL file or a C/C++ header for the native declarations of the COM interfaces. If you decide to write an IDL file, you can compile it with the Visual C++ SDK's MIDL compiler to produce a TLB.
```

Comment:
Let's also include links and guidance/example on how to update the SxS manifest.

Reply:
What changes need to be made to the SxS manifest to look up the tlb correctly? I personally don't actually know what's needed.

Reply:
https://docs.microsoft.com/windows/win32/sbscs/assembly-manifests

> The typelib element has the attributes shown in the following table.

Reply:
Do we also have a link for `COM Type Library (TLB)`?

Reply:
Also, `.NET Core assembly` -> `.NET assembly`?

Reply:
For the SxS manifest changes, we decided to include this in an additional PR.